v22.1/restore.md

```diff
@@ -92,6 +92,8 @@ You can restore:
 - [Databases](#databases)
 - [Tables](#tables)
 
+An object's historical data is not preserved upon `RESTORE`. This means that if you issue an `AS OF SYSTEM TIME` query on a restored object, the query will fail or the response will be incorrect because there is no historical data to query. For example, if you restore a table at `2022-07-13 10:38:00`, it is not then possible to read or [back up](backup.html) that table at `2022-07-13 10:37:00` or earlier. This is also the case for backups with [`revision_history`](backup.html#with-revision-history) that might try to initiate a revision start time earlier than `2022-07-13 10:38:00`.
+
```

Comment:
nit: I suspect a casual reader doesn't know what the term "historical data" means. Could it be worth saying something along the lines of: 

RESTORE will only restore the latest value of each key, or  the latest value AS OF SYSTEM TIME, if specified. In other words, historical data will not be restored even if the backup was specified with revision history.  

One consequence of this is that you cannot issue an AS OF SYSTEM TIME query on a restored object, the query will fail or the response will be incorrect because there is no historical data to query. For example, if you restore a table at `2022-07-13 10:38:00`, it is not then possible to read or [back up](backup.html) that table at `2022-07-13 10:37:00` or earlier.

Reply:
That's definitely something to consider and I'll adjust to make that a little more clear. We do use "historical data" quite often through our docs as part of backup, restore, and AS OF SYSTEM TIME docs. 

I think I'll need to reframe "key" in your example here to something more generic like data (or something more specific like a table in an example). We haven't typically referred to keys in this way, unless in arch docs. 

I'll make some adjustments and push another commit shortly 👍